### ISSUE TITLE
Couldn't fetch DOMElement

### ISSUE THREAD

**Issue by WindowsAndLinux** [punkave/phpQuery#8]
Couldn't fetch DOMElement. Node no longer exists in .../phpQuery/phpQuery.php on line 146

**Comment by wottpal** [comment]
Hey did you found a way to resolve this?

**Comment by WindowsAndLinux** [comment]
Nope, it's been over a year, and I worked on so many projects since then, that I honestly can't even remember why or where I wanted to use it.

**Comment by wottpal** [comment]
OK :) But thanks for your fast answer anyway!!
